Source/DOM classes/Unported or Partial DOM/SVGLineElement.m

```diff
@@ -41,6 +52,10 @@ -(CALayer *)newLayer
 	CGPathAddLineToPoint(path, NULL, _x2, _y2);
 	
 	CALayer* result = [SVGHelperUtilities newCALayerForPathBasedSVGElement:self withPath:path];
+	if (CGRectIsEmpty(result.bounds)) // empty rects don't get rendered
```

Comment:
Also: if a line is rendered from (0,0) to (0,0) ... what happens?

Appears to be a bug in the SVG spec: I can't find any statement on this :).

In the code you've got here, we'd generate a 1x1 bounding box, and potentiall render a width-wide pixel dot. Since the spec appears incomplete, I'd say that's OK?